media-video/qgifer/Manifest

```diff
@@ -1 +1,7 @@
+AUX qgifer-0.2.1-desktop.patch 526 SHA256 dd0a723dd7fb4479f1a5dfc927d41681aa43d6ef20ed180a938cfddb1e5ef57a SHA512 a5b58befb04339fc871e5cb33503996f536275da749ca6211d88ad519900a949c19883b646954c378d6bacaad0a43befbdeedf5b0504af9f82ea9483975fa69a WHIRLPOOL a6d28f537506136fc9f3432c345da19a2edba909980f126c32c2211f0c3610b433b73180f8cdc6956a6cb7ada03cfd70a04841cc45329f45339749073052cd13
+AUX qgifer-0.2.1-giflib5.patch 5309 SHA256 fba81387de60c8835c344488f069c58618565fdcf149c392e3133e9bff4b3075 SHA512 8d60f905335d98e21239cd8c24448a832ce196455410d4778ba861089353b393f18de76b232ec519ec21f66bd9cb0e6e24e72ddb75abb771b93609528cfd0063 WHIRLPOOL cfe179b751c37bb7c1f18919b3c1fa328078662961c31c04d97103416349e568ace0fabc0d9f29c768b4c1312d6fd2ad204a57657be6cda7e97728c88b8128c8
+AUX qgifer-0.2.1-giflib_5.0.6_to_5.1.0.patch 401 SHA256 e05a4c292617b37dbe2adc93be15d12d30b4639c89d51c894b73660f02e8b477 SHA512 c708d1dd9a40124468a13778c6a090e9ec04af192894a211a4993ddab6cf1acc3a7524323af8ab901fb9878a87632118337d4b1f8de1cd16a9121904af79a8ac WHIRLPOOL 653689dde306579df8530388d93ad8746fc85e1a66302aa0c0157a6cac36ee34f37d6be3155785dcd1a1911db9c2206f992f6c78e3f9f4cfddbf2cc5ce4a11f5
+AUX qgifer-0.2.1-opencv3.patch 2516 SHA256 3c558c5a612454a8ace78580713cd2697f22886221c850f236588b3aca5623a1 SHA512 95fc605f034a1bd9177dadc0a53505073b253e8b3265b7f07552212b0920225604946f5b711260a9e463e6926795f0867dbb8bfa41bdfd7ce3c247166e5c905a WHIRLPOOL cd1499d9a4d8db5c066fda72890cbb34b064efed87dfce27297bdd087aa0343fac0ed5a0a69f78e26dd072629c449aff9f9fbd7e2e27646bed75402810c53bcb
 DIST qgifer-0.2.1-source.tar.gz 373441 SHA256 b05f2b44faad2a94a746eb7de411875f133e6038ec81a5f3247c86a25a41e9f0 SHA512 ce48e5756cbab283d5894ddb369114e52c9cedc2fe15aed5d22aeeacc822058123d3a2f3c9a6eddf94a1eef992c3ddcad77d6941c76ba0c795c8b44209fac7df WHIRLPOOL 4095c8a6a373b234007b3015cef9ce9d4a023cf80e2701cec7ad5b2df1ab51e28cf63becb412d25e7305571e284b332542b8e15a7d49bd2d40ffe239753be5e4
+EBUILD qgifer-0.2.1-r5.ebuild 1099 SHA256 0eacb4226b595687e48c452e1e214b6b325ba7ecdafd0f554779407009d683ea SHA512 9f483d843a32c929086b2a620b07eac5c831d638054919be871450bbe951c3a56baf89ba1aa243a2ece534b3e5d9d26fda19601872a7399040dd52c3c0b23b46 WHIRLPOOL 8c25864571ccf3ce60387332f7f812ebd57029d65d06c7f639ecb98f929ba1f6ef9d6772188c80924c70ed8140e14e007ff94f38a975bffb23225fed22ba3e80
+MISC metadata.xml 661 SHA256 7655e8668e7a47aab71ed16ffc79456112bba988acc2f67e05b8b0490feeb5d5 SHA512 71689d9ad47bfcd52ce9334dc84cbcd8696cc1896d16a13ecd5f45a2d76fff68a59791c6d08b623549ad28610daed000dcd4ebd7f8d5023cc139753e6614f8ff WHIRLPOOL 953398c12acef5172d5305ed50f574cf6957d8f0cf519f5840297df0a627b73dcd472f49677d66d0f1d16eb39effc7632b8046dcb3a905ac9981a68583dc395c
```

Comment:
this is a fat manifest. The gentoo tree itself uses thin manifests. Use `repoman manifest` to generate thin manifests